docs/docsite/rst/release_and_maintenance.rst

```diff
@@ -13,44 +13,46 @@ Ansible is developed and released on a flexible 4 months release cycle.
 This cycle can be extended in order to allow for larger changes to be properly
 implemented and tested before a new release is made available.
 
-Ansible supports the two most recent major stable releases.
-For more information, read about the
-`development and stable version maintenance workflow`_.
+Ansible has a graduated support structure that extends to three major releases.
+For more information, read about the `development and stable version maintenance workflow`_. or see
+the chart in :ref:`schedule` for the degrees to which current releases are supported.
+
+.. note:: Support for three major releases began with Ansible-2.4.  Ansible-2.3 and previous are only
```

Comment:
Please change to “Ansible version 2.3 and older versions are only...”.  Also: one space after the end of a sentence, please - not two.